src/Microsoft.Framework.PackageManager/Sources/ListSourcesCommand.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Linq;
+
+namespace Microsoft.Framework.PackageManager
+{
+    internal class ListSourcesCommand
+    {
+        public Reports Reports { get; set; }
+        public string TargetDirectory { get; set; }
+
+        public int Execute()
+        {
+            // Most of this code is from NuGet, just some refactoring for our system.
+            // Collect config
+            var config = NuGetConfig.ForSolution(TargetDirectory);
+
+            var sources = config.Sources.LoadPackageSources().ToList();
```

Comment:
Probably not, I just need to add a counter. I copied this from `nuget sources` so between this and the below comment I'll probably just rework this a bit in general and I'll cover this comment there.